tests/core/test_push.py

```diff
@@ -1141,3 +1141,147 @@ async def test_global_topic_after_deleting_user(client, app):
         item["guid"], [topic], list(users.values()), companies
     )
     assert matching == {topic_id}
+
+
+# STT-51
+async def test_planning_to_many_events(client, app):
```

Comment:
Can we also have tests to cover this scenario:
* Push 1 Event
* Push 2 Planning items (that have a link to the same event)

Test:
* The list of Event coverages contains all coverages from both Planning 1 & 2, and does not contain any duplicates